bootstrap/sql/com.mysql.cj.jdbc.Driver/v013__create_db_connection_info.sql

```diff
@@ -209,3 +209,15 @@ ALTER TABLE team_entity DROP KEY `name`, ADD COLUMN nameHash VARCHAR(256) NOT NU
 ALTER TABLE user_entity DROP KEY `name`, ADD COLUMN nameHash VARCHAR(256) NOT NULL, ADD UNIQUE (nameHash);
 ALTER TABLE bot_entity DROP KEY `name`, ADD COLUMN nameHash VARCHAR(256) NOT NULL, ADD UNIQUE (nameHash);
 ALTER TABLE glossary_entity DROP KEY `name`, ADD COLUMN nameHash VARCHAR(256) NOT NULL, ADD UNIQUE (nameHash);
+
+-- Rename sourceUrl in pipeline_entity from DatabricksPipeline & Fivetran
+UPDATE pipeline_entity
+SET json = JSON_REMOVE(json, '$.sourceUrl')
```

Comment:
that's not a rename right?

Reply:
oh sorry, it should be remove instead of rename

Reply:
Have removed this field because we were storing empty string in these connectors and empty string won't be a valid url format

Reply:
makes sense, thanks